indra/llcharacter/llcharacter.h

```diff
@@ -119,6 +119,8 @@ class LLCharacter
 
 	virtual void addDebugText( const std::string& text ) = 0;
 
+	virtual std::string getSecureName() const { return getID().asString(); }
```

Comment:
I feel like calling it 'secure name' is a misnormer...

Reply:
I second this. What does "secure" mean in this case? I see just an ID.

Reply:
It's overriden below, so it isn't just an id, it can return an actual name.

Purpose of this function is purely for debug, so I suggest using getDebugName or getDebInfoName given the definition check.